convert.js

```diff
@@ -6,8 +6,8 @@ function run(){
     let uri = URI(text_in.value);
     let new_uri = new URI({
       protocol: "https",
-      hostname: "easyroom.unitn.it",
-      path: "/Orario/ec_download_ical_list.php",
+      hostname: "easyacademy.unitn.it",
+      path: "/AgendaStudentiUnitn//ec_download_ical_list.php",
```

Comment:
C'è una slash in più
```suggestion
      path: "/AgendaStudentiUnitn/ec_download_ical_list.php",
```

Reply:
è voluto il doppio slash

Reply:
Cioè senza non funziona? Wow. Easyroom/academy non finisce mai di stupirmi.